src/core/openthread-core-default-config.h

```diff
@@ -914,10 +914,20 @@
 #define OPENTHREAD_CONFIG_NUM_DHCP_PREFIXES 4
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_ENABLE_SLAAC
+ *
+ * Define as 1 to enable support for adding of auto-configured SLAAC addresses by OpenThread.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_ENABLE_SLAAC
+#define OPENTHREAD_CONFIG_ENABLE_SLAAC 1
```

Comment:
merge this with `OPENTHREAD_CONFIG_NUM_SLAAC_ADDRESSES`, or define as
`OPENTHREAD_CONFIG_NUM_SLAAC_ADDRESSES == 0` ?

Reply:
Not sure. Don't think we use this model for DHCP or any other case. I feel `ENABLE_SLAAC` is more explicit about intention. 

Was actually considering if it makes sense to use `configure.ac` option for SLAAC instead of header file one. 
Thoughts?

Reply:
Sounds good to me.